index.html

```diff
@@ -11,11 +11,21 @@
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
 </head>
 <body>
-    <header>
+    <div>
         <nav class="hdr">
-            <div class="logo"><img src="./images/MyLogo.png" alt="My Logo"></div>
+            <div class="logo">
+                <p>My Logo</p>
+            </div>
             <div class="bar"><img src="./images/Union.png" alt="menu"></div>
+            <div class="bar-desktop">
+                <ul class="bar-desktop-list">
+                    <li class="bdl-el">Portfolio</li>
+                    <li class="bdl-el">About</li>
+                    <li class="bdl-el">Contact</li>
+                </ul>
+            </div>
```

Comment:
- [ ] [OPTIONAL] kindly add for each list item a link tag, since they are supposed to be links to navigate through the website